hubstorage/resourcetype.py

```diff
@@ -18,6 +19,23 @@ def __init__(self, client, key, auth=None):
         self.auth = xauth(auth) or client.auth
         self.url = urlpathjoin(client.endpoint, self.key)
 
+    @property
+    def _supports_msgpack(self):
+        return self.resource_type in ['items', 'logs',
+                                      'collections', 'samples']
+
+    @staticmethod
+    def _enforce_msgpack(**kwargs):
+        kwargs['stream'] = True
```

Comment:
I see that `_iter_lines` doesn't enforce `stream=True`, and `iter_json` below - does. Any reason why we don't follow this approach with msgpack?